masterlist.yaml

```
@@ -8820,6 +8820,19 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507'
+        name: 'Nexus Mods'
```

The location name should be the name of the mod's page, not just the website it's hosted on. That should only be appended if multiple links with the same name are provided:

```suggestion
        name: 'Woodworker''s Whim'
```